doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -0,0 +1,604 @@
+
+#. Insert the Ubuntu USB boot disk into the WHL Maxtang machine.
+#. Power on the machine, then press F11 to select the USB disk as the boot
+   device. Select **UEFI: SanDisk**. Note that the label depends on the brand/make of the USB stick.
+#. Install the Ubuntu OS.
+#. Select **Something else** to create the partition.
+
+   .. figure:: images/native-ubuntu-on-SATA-1.png
+
+#. Configure the ``/dev/sda`` partition. Refer to the diagram below:
+
+   .. figure:: images/native-ubuntu-on-SATA-2.png
+
+   a. Select the ``/dev/sda`` partition, not ``/dev/nvme0p1``.
+   b. Select ``/dev/sda`` **ATA KINGSTON RBUSNS4** as the device for the
+      boot loader installation. Note that the label depends on the on the SATA disk used.
+
+#. Continue with the Ubuntu Service VM installation in ``/dev/sda``.
+
+Install the Ubuntu Service VM on the NVMe disk
+----------------------------------------------
+
+.. note:: Before you install the Ubuntu Service VM on the NVMe disk, either
+   remove the SATA disk or disable it in the BIOS. Disable it by going to:
+   **Chipset** → **PCH-IO Configuration** -> **SATA and RST Configuration** -> **SATA Controller [Disabled]**
+
+#. Insert the Ubuntu USB boot disk into the WHL Maxtang machine.
+#. Power on the machine, then press F11 to select the USB disk as the boot
+   device. Select **UEFI: SanDisk**. Note that the label depends on the brand/make of the USB stick.
+#. Install the Ubuntu OS.
+#. Select **Something else** to create the partition.
+
+   .. figure:: images/native-ubuntu-on-NVME-1.png
+
+#. Configure the ``/dev/nvme0n1`` partition. Refer to the diagram below:
+
+   .. figure:: images/native-ubuntu-on-NVME-2.png
+
+   a. Select the ``/dev/nvme0n1`` partition, not ``/dev/sda``.
+   b. Select ``/dev/nvme0n1`` **FORESEE 256GB SSD** as the device for the
+      boot loader installation. Note that the label depends on the on the NVMe disk used.
+
+#. Complete the Ubuntu installation and reboot the system.
+
+   .. note:: Set **acrn** as the username for the Ubuntu Service VM.
+
+
+Build and Install ACRN on Ubuntu
+********************************
+
+Pre-Steps
+=========
+
+#. Set the network configuration, proxy, etc.
+#. Update Ubuntu:
+
+   .. code-block:: none
+
+      $ sudo -E apt update
+
+#. Create a work folder:
+
+   .. code-block:: none
+
+      $ mkdir /home/acrn/work
+
+Build the ACRN Hypervisor on Ubuntu
+===================================
+
+#. Install the necessary libraries:
+
+   .. code-block:: none
+
+      $ sudo -E apt install gcc \
+        git \
+        make \
+        gnu-efi \
+        libssl-dev \
+        libpciaccess-dev \
+        uuid-dev \
+        libsystemd-dev \
+        libevent-dev \
+        libxml2-dev \
+        libusb-1.0-0-dev \
+        python3 \
+        python3-pip \
+        libblkid-dev \
+        e2fslibs-dev \
+        pkg-config \
+        zlib1g-dev \
+        libnuma-dev \
+
+        liblz4-tool
```

Comment:
@fuzhongl Can you answer this?

Reply:
It is the input of Gray Zhang from IOTG, let me check with him offline; then update the comments.
Thanks!

Reply:
@gvancuts  Double checked with @fyin1 
zlib1g-dev isn't need anymore since following PR is merged into release 2.0
https://github.com/projectacrn/acrn-hypervisor/pull/4839

liblz4-tool is needed to build kernel